CHANGES

```diff
@@ -59,6 +59,33 @@ Standard Library
 
 - Added `Ndigits.N2Bv_sized`, and proved some lemmas about it.
 
+- The scopes `int_scope` and `uint_scope` have been renamed to
+  `dec_int_scope` and `dec_uint_scope`, to clash less with ssreflect
+  and other packages.  They are still delimited by `%int` and `%uint`.
+
+- Syntax notations for `string`, `ascii`, `Z`, `positive`, `N`, `R`,
+  and `int31` are no longer available merely by `Require`ing the files
+  that define the inductives.  You must `Import` `Coq.Strings.String`,
+  `Coq.Strings.Ascii`, `Coq.ZArith.BinIntDef`, `Coq.PArith.BinPosDef`,
+  `Coq.NArith.BinNatDef`, `Coq.Reals.Rdefinitions`, and
+  `Coq.Numbers.Cyclic.Int31.Cyclic31`, respectively, to be able to use
+  these notations.  Note that passing `-compat 8.8` or issuing
+  `Require Import Coq.Compat.Coq88` will make these notations
+  available.  Users wishing to port their developments automatically
+  may download `fix.py` from [this
+  gist](https://gist.github.com/JasonGross/5d4558edf8f5c2c548a3d96c17820169)
```

Comment:
Given that CHANGES is not yet a Markdown file, I'd suggest to rather say:
```
may download `fix.py` from <https://gist.github.com/JasonGross/5d4558edf8f5c2c548a3d96c17820169>
and run a command like ...
```
This is also Markdown-compatible syntax but it is less weird in a text file.

Reply:
Done